puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/lib/ansible.groovy

```diff
@@ -43,3 +49,23 @@ def writeExtraVars(args) {
 
     return extra_vars_file
 }
+
+def buildExtraVars(args) {
+    def timestamp = System.currentTimeMillis()
+    def extra_vars_file = 'extra_vars-' + timestamp.toString()
```

Comment:
given it's a string, do we need to write it to file at all? as the file won't be used and the output will have the data anyways